services/start.sh

```diff
@@ -2,5 +2,6 @@
 
 composer install --prefer-source --no-interaction --no-dev --optimize-autoloader
 
-php artisan migrate --force
+# Disable migrations because they will now happen in Hasura.
```

Comment:
Have we setup Hasura to run all the migrations on an initial DB setup? I.e. from fresh PostgreSQL db to where we are now?

Reply:
> start.sh

yeap it isn't used at all

Reply:
Yes, Hasura is now the source of truth for migrations, or it needs to be because Zashton already has it running a migration for the private view of notes